src/hooks/delayedSelectors.ts

```diff
@@ -28,7 +29,10 @@ type ModeFromConf<C extends DSConfig> = C extends DSConfig<any, infer M> ? M : n
  * will not be able to be used as a cache key.
  */
 export function useDelayedSelector<C extends DSConfig>(props: DSProps<C>): DSReturn<C> {
-  const state = useRef(new SingleDelayedSelectorController(props));
+  const state = useRef<SingleDelayedSelectorController<C>>();
+  if (!state.current) {
+    state.current = new SingleDelayedSelectorController(props);
+  }
```

Comment:
What difference does this make? 🤔

Reply:
Oh, I never thought about that, but of course it does, its just a function parameter after all 🤦